spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+RSpec.describe "Show spec" do
+  before(:each) do
+    @emma = Hotel.create!(name: "Hotel Emma", location: "San Antonio")
+    @wolf = Hotel.create!(name: "Great Wolf Lodge", location: "Dallas")
+    
+    @room_1 = @emma.rooms.create!(rate: 500, suite: "Presidential")
+    @room_2 = @emma.rooms.create!(rate: 650, suite: "Penthouse")
+    @room_3 = @wolf.rooms.create!(rate: 300, suite: "Garden")
+    @room_4 = @wolf.rooms.create!(rate: 400, suite: "Animal Kingdom")
+    
+    @sara = Guest.create!(name: "Sara", nights: 2)
+    @leo = Guest.create!(name: "Leo", nights: 3)
+    @piper = Guest.create!(name: "Piper", nights: 3)
+    @paige = Guest.create!(name: "Paige", nights: 1)
+
+    @room_4.guests << @sara
+    @room_1.guests << @sara
+    @room_2.guests << @paige
+    @room_1.guests << @paige
+  end
+  # user story 1
+  describe "As a visitor, when going to the guest show page" do
+    it "has the guests name and a list of all rooms they are in:shows room's suite, nightly rate, and the name of the hotel that it belongs to" do
+      visit "/guests/#{@paige.id}"
+
+      expect(page).to have_content(@paige.name)
```

Comment:
These assertions could be more robust if you verify the text is there as it is displayed in your view, so:
`expect(page).to have_content("Hotel Emma, Penthouse, Room Rate: 650")`
By doing this, you're making sure the dynamic parts of this page are populating properly and *in the right place* rather than floating randomly on the page